### DISCUSSION TITLE
AYUDA CON FECHA LÍMITE

### DISCUSSION THREAD

**Discussion by diegovillacrypto** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#386] [Q&A]
Que tal amigos buenos tardes, por favor necesito ayuda con la orientación de Front-end, siento todo revuelto con el seguimiento de los contenidos y quiero colocarme al día para poder entregarlo hoy.

Me voy a dedicar las horas necesarias tengo experiencia pero necesito saber cuál es la ruta. Muchas gracias

**Comment by doguedogue** [comment]
La ruta está en el teams dividido por semanas.

Cada una ha tenido una entrega, que por una u otra razón, luego dan un periodo adicional de entrega.

Hoy se entregaría css (aún no tenemos la forma para enviarla), quiere decir que ya pasamos por intro(maquetación) y HTML.

En esta semana que pasó terminó la semana 4 que fue sobre Javascript.

Igual puedes ver los vídeos si algo se te pasa.

![Screenshot_2022-03-13-13-59-36-721_com.microsoft.teams.jpg](https://user-images.githubusercontent.com/23409026/158076809-4063268c-4f3e-45c8-b5d1-5cf8a6e3854d.jpg)

**Reply by diegovillacrypto** [reply]
Amigo muchas gracias, pero todavía tengo chance de hacer las entregas de las primeras semanas?

**Reply by chris-amaya** [reply]
la verdad pregunta al correo de ayuda de inovaccion por que mandaron correos diciendo esto, dandome la sensación que ya no se puede enviar practicas anteriores :c 

![image](https://user-images.githubusercontent.com/24614876/158084297-ee165276-ddd2-4d13-9878-8b30d44270c0.png)

**Reply by chris-amaya** [reply]
este es el correo
mailto:[contacto@innovaccion.mx](mailto:contacto@innovaccion.mx)
